javascripts/discourse/api-initializers/table-editor.js

```diff
@@ -59,7 +59,11 @@ export default apiInitializer("0.11.1", (api) => {
       if (table.parentNode.contains(expandBtn)) {
         expandBtn.parentNode.insertBefore(popupBtn, expandBtn);
       } else {
-        table.parentNode.insertBefore(popupBtn, table);
+        const buttonWrapper = document.createElement("div");
+        buttonWrapper.classList.add("fullscreen-table-wrapper-buttons");
+        buttonWrapper.append(popupBtn);
+        popupBtn.addEventListener("click", generateModal, false);
```

Comment:
With this refactor, we're now adding two click event listeners to `popupBtn`: one inside this `else` clause, and a second one below (line 69): 

```
popupBtn.addEventListener("click", generateModal.bind(attrs), false);
```

I also noticed that one is binding attributes but the other one isn't.